### DISCUSSION TITLE
Span footer full width

### DISCUSSION THREAD

**Discussion by karani12** [vuepress/core#776] [Q&A] [answered]
is there a way I could make the footer on homepage and content page span the whole width in the default theme?

**Comment by yzqdev** [comment, marked as answer]
Maybe you could set something like `--content-width: 100%`, or use your own css to change the interface.

Here's the document page https://v2.vuepress.vuejs.org/reference/default-theme/styles.html#style-file. Hope this could help you

**Comment by Mister-Hope** [comment]
> is there a way I could make the footer on homepage and content page span the whole width in the default theme?

I am not sure what you want, maybe you should explain with more details.
